modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -90,17 +227,25 @@ public function add_plugin_menus() {
 	}
 
 	public function enqueue() {
-		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'calypsoify/style.css', false, JETPACK__VERSION );
+		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'style.min.css', false, JETPACK__VERSION );
```

Comment:
What would you say about using `Jetpack::get_file_url_for_environment` for those minified files, so we can get both the minified or full versions when needed?

Reply:
From what I saw it didn't seem as though the build tools were generating unminified versions of the non-rtl css files?  I'll recheck to be sure.

Reply:
Yeah, it's not generating unminified versions of the css files -- there's just the `scss` and `-min`